src/components/shared/Charts/BarChart/index.js

```diff
@@ -0,0 +1,72 @@
+// React
+import React from 'react';
+
+// Styled
+import {
+  ChartWrapper,
+  Bar,
+  Chart,
+  HLines,
+  HLine,
+  YAxisLabel,
+  XAxisLabels,
+} from './styled';
+
+export default ({ data }) => {
+  const dataLength = data.length;
+  const mostValue = data.reduce((acc, curr) => {
+    const { value } = curr;
+    return value > acc ? value : acc;
+  }, 0);
+  const magnitude = 10 ** (mostValue.toString().length - 1);
+  const segmentation = Math.floor(mostValue / magnitude + 1);
+  const YAxisMax = segmentation * magnitude;
+  const YAxisSegm = [...Array(Math.ceil((segmentation + 1))).keys()].filter((element, index) => index % 2 === 0);
```

Comment:
```javascript
const YAxisSegm = new Array(Math.ceil((segmentation + 1))).fill(null).filter((_, index) => index % 2 === 0);
```

Reply:
doesn't work, get an array filled null value.
what did you mean? why this way?

Reply:
```suggestion
.filter((element, index) => index % 2 === 0);
```
so when i need only one of all params, I should name the unnecessary variable like that: "_". Am I right?

Reply:
Oh, sorry.

Just replace `index % 2 === 0` with `!(index % 2)`

And about "`_`" - yes, correct.

Reply:
It's still not working. I getting the same result. Why should I use filter for array filled null value? It makes no sense.

Reply:
No, I mean that you can use your solution but with `!(index % 2)` instead of `index % 2 === 0`